Sources/NIOPriorityQueue/Heap.swift

```diff
@@ -183,6 +183,7 @@ internal struct Heap<T: Comparable> {
 
 @available(*, deprecated, message: "The NIOPriorityQueue module is deprecated and will be removed in the next major release.")
 extension Heap: CustomDebugStringConvertible {
+    @available(*, deprecated, message: "The NIOPriorityQueue module is deprecated and will be removed in the next major release.")
```

Comment:
I assume these were warnings in 4.2? What is the alternative to the `NIOPriorityQueue`, is that something that can be added?

Reply:
@BasThomas we deprecated the old one and moved it to NIO for performance reasons.

Reply:
Yes, but I was wondering what "the old ones" are. If they are mentioned, it would make a migration easier.

In an `@available`, you can then use the `renamed` parameter to provide users a fix-it within Xcode.

Reply:
There is no migration: they have been made non-public. Their publicity was accidental only.